pkg/storaged/content-views.jsx

```diff
@@ -905,6 +905,7 @@ export class VGroup extends React.Component {
                     Message(cockpit.format(_("The $0 package will be installed to create VDO devices."), vdo_package),
                             {
                                 visible: vals => vals.purpose === 'vdo' && need_vdo_install,
+                                style: { marginTop: '1px' }
```

Comment:
This change is unrelated and should not be in this PR.

Reply:
Ya i think its an old change and don't know how it got committed when I only added one file to commit.